.goreleaser.yaml

```diff
@@ -47,6 +47,28 @@ builds:
       - 'amd64'
       - 'arm64'
 
+  -
+    id: lumberctl
```

Comment:
Quote your YAML plz

Also, do we have `yamllint` installed on this repo?

Reply:
Don't think so, I will open a separate PR to add that, thx!